.github/workflows/ui-tests-critical.yml

```diff
@@ -83,17 +113,21 @@ jobs:
         run: |
           maestro test ${{env.MAESTRO_FLOWS_PATH}} --format junit --debug-output ${{env.MAESTRO_LOGS_PATH}}
 
+      - name: Delete Simulator
+        if: always()
+        run: ./scripts/delete-simulator.sh
```

Comment:
`l`: why do we have to delete the simulator? Wouldn't it get discarded anyways by the CI runner?

Reply:
Yes, in theory GH should remove it.